shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/SchemaBuilder.java

```diff
@@ -126,38 +144,38 @@ private static void appendDialectRemainTables(final DialectTableMetaDataLoader d
             }
         }
     }
-    
-    private static void appendDefaultRemainTables(final SchemaBuilderMaterials materials, final ShardingSphereSchema schema) throws SQLException {
-        Collection<String> existedTableNames = getExistedTables(materials.getRules(), schema);
-        for (Entry<String, DataSource> entry : materials.getDataSourceMap().entrySet()) {
+
+    private static void appendDefaultRemainTables(final SchemaBuilderMaterials materials, final Map<String, TableMetaData> tables) throws SQLException {
+        Collection<String> existedTableNames = getExistedTables(materials.getRules(), tables);
+        for (Map.Entry<String, DataSource> entry : materials.getDataSourceMap().entrySet()) {
```

Comment:
Please import `Map.Entry` to avoid `Map.`

Reply:
done